tensorboard/plugins/histogram/tf_histogram_dashboard/histogramCore.ts

```diff
@@ -107,12 +114,21 @@ export function intermediateToD3(
       // infinite-sized.
       const bucketRight = Math.min(max, histogram.buckets[bucketIndex].right);
       const bucketLeft = Math.max(min, histogram.buckets[bucketIndex].left);
-      const intersect =
-        Math.min(bucketRight, binRight) - Math.max(bucketLeft, binLeft);
-      const count =
-        (intersect / (bucketRight - bucketLeft)) *
-        histogram.buckets[bucketIndex].count;
-      binY += intersect > 0 ? count : 0;
+      const bucketWidth = bucketRight - bucketLeft;
+      if (bucketWidth > 0) {
+        const intersect =
+          Math.min(bucketRight, binRight) - Math.max(bucketLeft, binLeft);
+        const count =
+          (intersect / (bucketRight - bucketLeft)) *
+          histogram.buckets[bucketIndex].count;
+        binY += intersect > 0 ? count : 0;
+      } else {
+        const isFinalBin = binRight >= max;
+        const singleValueOverlap =
+          binLeft <= bucketLeft &&
+          (isFinalBin ? bucketRight <= binRight : bucketRight < binRight);
+        binY += singleValueOverlap ? histogram.buckets[bucketIndex].count : 0;
+      }
```

Comment:
Is this change important for fixing the build breakage or is this something else? If it is related, would you mind telling me how this is related?

Reply:
This was mixed with another branch for some reason. Sorry for the confusion!